packages/calypso-e2e/src/lib/pages/cart-checkout-page.ts

```diff
@@ -233,6 +256,15 @@ export class CartCheckoutPage {
 		await cvvInput.fill( paymentDetails.cvv );
 	}
 
+	/**
+	 * Enter cardholder's name in the payment details form.
+	 *
+	 * @param {PaymentDetails} paymentDetails Object implementing the PaymentDetails interface.
+	 */
+	async enterCardholderName( paymentDetails: PaymentDetails ): Promise< void > {
+		await this.page.fill( selectors.cardholderName, paymentDetails.cardHolder );
```

Comment:
Quick question - I assume the method `CartCheckoutPage.enterPaymentDetails` is not sufficient for this?

Reply:
`CartCheckoutPage.enterPaymentDetails` should be enough - I've modified the code to use this instead.